frameworks/cassandra/RELEASE_NOTES.md

```diff
@@ -0,0 +1,50 @@
+### Package Upgrade in Strict Mode
+
+If you are upgrading a service running in `strict` mode to the new package version (this version), you need to specify service account details as options during the package update process.
+
+
+You need to give service account details in options in order to authenticate and run in DC/OS `strict` mode. Or you set them in the GUI. Here is how a service is installed using a previous package version:
+
```

Comment:
"You can provide service account details in a JSON options file or via the DC/OS GUI."

Is it helpful to show a user the old config? If it's not crucial, I suggest removing it. Users like to cut-and-paste and could end up with exactly the wrong configuration.